ES/ESmy018.xml

```diff
@@ -32,86 +32,185 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                     me and bless us, the sinful, in the time of incense and communion. [fol.74ra]
-                     Your intercession, your intercession do not forget for us! The part of the
-                     colophon mentioning Zabana Krǝstos was added later. </colophon>
+                     was added later.
+                   </colophon>
+
+                    </msItem>
+                     <msItem xml:id="ms_i4">
+                       <locus from="44ra" to="60va"/>
+                       <title type="complete" ref="LIT2085OntheL">Discourse by John Chrysostom on the life and
+                         decollation of John the Baptist</title>
+                         <incipit xml:lang="gez">
+                            <locus target="#44ra"/><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወ</hi>መንፈስ፡ ቅዱስ፡ አሐዱ፡ <hi rend="rubric">አምላክ፡
+                            ዝንቱ፡ ነገር፡ በከ</hi>መ፡ ኮነ፡ ሕይወቱ፡ ለቅዱስ፡ <hi rend="rubric">ዮሐንስ፡ መጥምቅ፡ ወከ</hi>መ፡ እፎ፡
+                            ኮነ፡ ምትረተ፡ ርእሱ፡ ወተርጐሞ፡ ቅዱስ፡ ዮሐንስ፡ አፈ፡ ወርቅ፡ በትርያርክ፡ ሊቀ፡ ጳጳሳት፡ ብሂል፡ ዘሀገረ፡ ቈስጠንጥንያ፤ ወሶበ፡ ኮነ፡ ስደቱ፡ ውስተ፡ አብራክይስ፡ አመ፡ ተስእሎ፡ አይሁዳዊ፡
+                            ኬልቄዶናዊ፡ ዘአምነ፡ በእግዚእነ፡ ኢየሱስ፡ ክርስቶስ።
+                         </incipit>
+
+                     <msItem xml:id="ms_i7">
+                      <locus from="60vb" to="63ra"/>
+                      <title type="complete" ref="LIT2375Synaxa#30YakkatitJohnBaptist">On the
+                        finding of the head of John the Baptist</title>
+                      </msItem>
+
+                      <msItem xml:id="ms_i8">
+                       <locus from="63ra" to="65ra"/>
+                       <title type="complete" ref="LIT2375Synaxa#2SaneJohnBaptist">On the finding of the
+                         body of John the Baptist</title>
+                       </msItem>
+
+                       <msItem xml:id="ms_i9">
+                        <locus from="65ra" to="74ra"/>
+                        <title type="complete" ref="LIT2125OntheG">Homily by John
+                          Chrisostom on the glory of John the Baptist</title>
+                        <incipit xml:lang="gez">
+                        <locus from="65ra" to="65rb"/>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ ድርሳን፡ ዘደረሰ፡ ዮሐንስ፡ አፈ፡ ወርቅ፡
+                        ካህን፡ ሊቀ፡ ጳጳሳት፡ ዘቍስጥንጥንያ፡ በእንተ፡ ክብሩ፡ ለዮሐንስ፡ መጥምቅ፡ በከመ፡ ጽሐፍ፡ ወስተ፡ ወንጌል፡ ዘማቴዎስ፡ ምንተኑመ፡ ወፃእክሙ፡
+                        ትርአዩ፡ ገዳመ፡ ተናገረ፡ ወይቤ፡ በእንተ፡ ሰማያት፡ ፫ዘወሀቦ፡ እግዚአብሔር፡ ለዮሐንስ፡ መጥምቅ፡ እም፴፡ ለሰኔ፡ ዘይትነበብ፡ በሰላመ፡ እግዚአብሔር፡
+                        አሜን፡ ስምዑ፡ ኦፍቁራንየ፡ አንሰ፡ እፈቅድ፡ በእንተ፡ ዕበዩ፡ ለዮሐንስ፡
+                      </incipit>
+                        <note>The text is not separated from the previous <ref target="#ms_i9"/>.</note>
+                        </msItem>
+
+                        <colophon xml:id="coloph2">
+                          <locus from="74ra" to="74rb"/>
+                         <foreign xml:lang="gez">ተፈጸመ፡ በዝየ፡ መጽሐፈ፡ መልእክት፡ ዘቅዱስ፡ ዮሐንስ፡ መጥምቀ፡
```

Comment:
remove the foreign element around the geez text

Reply:
ok

Reply:
Sure? Is it a mistake? In DS we did it the way I have done here.

Reply:
you do not need it if you add the xml:lang to the colophon tag, the way you do with the incipits